db/seeds.rb

```diff
@@ -27,3 +30,34 @@
 item16 = coffee.items.create!(title: 'Cocoa', description: 'delicious organic chocolate to be savoured as it is or combined with intriguing infusions.', price: 500, image: File.new("#{Rails.root}/app/assets/images/coco.jpg"))
 item17 = coffee.items.create!(title: 'Latte', description: 'A shot of strong espresso coffee, with a healthy covering of hot steamed milk and a topping of steamed milk froth served in a glass.', price: 500, image: File.new("#{Rails.root}/app/assets/images/latte.jpg"))
 item18 = coffee.items.create!(title: 'Green Tea', description: 'Green tea potential health benefits for everything from fighting cancer to helping your heart, and it taste pretty okay too.', price: 500, image: File.new("#{Rails.root}/app/assets/images/green-tea.jpg"))
+
+
+jeff_order_1 = item1.orders.create(total_cost: item1.price, user_id: jeff.id, status: 'completed')
```

Comment:
Line is too long. [98/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.